items.py

```diff
@@ -0,0 +1,54 @@
+class item():
```

Comment:
All these items also need class methods! Right now, they just have states. So think about what the items can do or have done to them. For example use_taser(player) could affect the players health based on the taser damage. Something like that!

You need at least two methods for your objects to meet the requirements!